client/app/organization/[id]/page.tsx

```
@@ -0,0 +1,629 @@
+
+                            <div className="flex flex-wrap gap-2">
+                              {turf.sports &&
+                                turf.sports.map((sport) => (
+                                  <Badge
+                                    key={sport}
+                                    variant="secondary"
+                                    className="bg-green-50 text-green-600 text-xs px-2 py-0.5 rounded-md"
+                                  >
+                                    {sport}
+                                  </Badge>
+                                ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)